src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -180,7 +243,20 @@ public override bool TryGetPropertyValue(string name, out object value)
         {
             if (name == null)
             {
-                throw Error.ArgumentNull(nameof(name));
+                throw Error.ArgumentNull("name");
```

Comment:
```suggestion
                throw Error.ArgumentNull(nameof(name));
```

Reply:
In this case we don't need to check if string is empty like in the `TrySetPropertyValue` case?